resources/featherlight.scss

```diff
@@ -0,0 +1,91 @@
+@import "vars";
+
+// Based on Featherlight 1.7.14
+
+.featherlight {
+    display: none;
+    position: fixed;
+    top: 0; right: 0; bottom: 0; left: 0;
+    z-index: 2147483647;
+    text-align: center;
+    white-space: nowrap;
+    cursor: pointer;
+    background: #333;
+
+    &:last-of-type {
+        background: rgba(0, 0, 0, 0.8);
+    }
+
+    &:before {
+        content: '';
+        display: inline-block;
+        height: 100%;
+        vertical-align: middle;
+    }
+
+    & .featherlight-content {
```

Comment:
Why do you need `& .`?

Reply:
Featherlight used `.featherlight .featherlight-content`, and I wrote it here as `& .featherlight-content`.

Reply:
This is redundant. `.featherlight-content` works just fine. It nests by default and `&` is only needed when it's on the same level.

Reply:
Ok, cleaned up.